File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio12.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio12 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o preço do produto (em reais):");
        float preco = entrada.nextFloat();

        float desconto = preco * 5/100;
        float novoPreco = preco - desconto;
        System.out.println("Preço promocional (-5%): R$ " +novoPreco);
    }
}

File: guanabara-c/lista01/exercicio12.c
#include <stdio.h>

int main(){
    float preco_atual, novo_preco, desconto;

    printf("Insira o preço do produto: ");
    scanf("%f", &preco_atual);

    desconto = preco_atual * 5 / 100;
    novo_preco = preco_atual - desconto;
    printf("Preço promocional: R$ %.2f", novo_preco);

    return 1;
}
